src/navigation/MainTabNavigator.tsx

```diff
@@ -9,101 +13,154 @@ import HomeStack from "./HomeStack"
 import SymptomHistoryStack from "./SymptomHistoryStack"
 import SettingsStack from "./SettingsStack"
 import { useConfigurationContext } from "../ConfigurationContext"
-
 import { Stacks } from "./index"
 import { TabBarIcons } from "../assets/svgs/TabBarNav"
-import { Colors } from "../styles"
+import { Text } from "../components"
 
-const Tab = createBottomTabNavigator()
+import { Colors, Layout, Outlines, Spacing, Typography } from "../styles"
 
 const MainTabNavigator: FunctionComponent = () => {
   const { t } = useTranslation()
   const insets = useSafeAreaInsets()
   const { displaySymptomHistory } = useConfigurationContext()
 
-  interface TabIconProps extends TabBarIconProps {
-    icon: string
+  interface TabBarIconProps {
+    focused: boolean
+  }
+
+  const HomeIcon: FunctionComponent<TabBarIconProps> = ({ focused }) => {
+    return <TabIcon icon={TabBarIcons.House} focused={focused} />
   }
 
-  const TabIcon: FunctionComponent<TabIconProps> = ({
+  const ExposureHistoryIcon: FunctionComponent<TabBarIconProps> = ({
     focused,
-    size,
-    icon,
   }) => {
+    const tabIcon = <TabIcon icon={TabBarIcons.Exposure} focused={focused} />
+    return tabIcon
+  }
+
+  const HeartbeatIcon: FunctionComponent<TabBarIconProps> = ({ focused }) => {
```

Comment:
thoughts on not extracting these functions and passing the Icon in as a prop to TabIcon?